src/features/Auth/AuthSlice.tsx

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from '@reduxjs/toolkit'
+import type { PayloadAction } from '@reduxjs/toolkit'
+import { statSync } from 'fs'
+
+
+export interface authState {
+  isLoggedIn: boolean
+}
+
+const initialState: authState = {
+  isLoggedIn: false //logic to check localStorage
+}
+
+export const authSlice = createSlice({
+  name: 'auth',
+  initialState,
+  //how state can be updated
+  reducers: {
+    setLoggedIn: (state, action) => {
+        state.isLoggedIn = action.payload;
+        console.log('testing 123 :', state.isLoggedIn);
+    }
+    // authPass: (state) => {
```

Comment:
It would be nice if you just remove the commented previous codes

Reply:
I have removed!